runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/graph/GreedyPipelineFuser.java

```diff
@@ -69,6 +69,15 @@ private GreedyPipelineFuser(Pipeline p) {
     fusePipeline(groupSiblings(rootConsumers));
   }
 
+  /**
+   * Fuses a {@link Pipeline} into a collection of {@link ExecutableStage}s.
```

Comment:
`{@link ExecutableStage ExecutableStages}` is our normal style.

Reply:
Done.